tests/integration/CreateSprintTemplate.test.ts

```diff
@@ -0,0 +1,143 @@
+
+dotenv.config();
+const username = process.env.TEST_FIXTURE_USERNAME;
+const password = process.env.TEST_FIXTURE_PASSWORD;
+
+const randomTemplateName = nanoid(10);
+describe("Login Page:", () => {
+  let browser: Browser;
+  let page: Page;
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch({
+      // headless: false, // <------------------------ This is the line that makes it so you can see whats going on
+
+      slowMo: 50, // <-------- use this if you want to slow down the tests so you can see what is happening, 50 minimum to make it work properly or its too fast.
+      defaultViewport: null,
+      args: ["--window-size=1400,1022"], // In headed mode you need the window size to have a width of at least 1400px or the drag test will fail as there isnt enough window space to drag the box to the correct location
+    });
+    page = await browser.newPage();
+    await installMouseHelper(page); // This is a helper function that allows you to see what the mouse is actually doing on the screen in headed mode.
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  it("should allow a user to log in and log out", async () => {
+    await page.goto(`http://localhost:5173/login`, {
+      timeout: 90_000,
+      waitUntil: "networkidle2",
+    });
+    await page.waitForSelector("#email");
+    await page.type("#email", username || "");
+    await page.type("#password", password || "");
+    await page.click("[type='submit']");
+    await page.waitForSelector(".sticky-logout", { timeout: 90_000 });
+    const logout = await page.$(".sticky-logout");
+    if (typeof logout === "undefined") {
+      await page.screenshot({ path: "login-failure.png" });
+    }
+    expect(typeof logout === "undefined").to.equal(false);
+  }, 90_000);
+
+  it("should allow a user to create a sprint template", async () => {
+    await page.waitForSelector(`[id="The Arena"]`, { visible: true });
+    await page.click(`[id="The Arena"]`);
+    console.log("clicking carrot");
+    await page.waitForSelector(`[id="Sprint Template"]`);
+    await page.click(`[id="Sprint Template"]`);
+    console.log("clicking sprint template");
+    await page.waitForSelector("#template-name");
+    await page.type("#template-name", randomTemplateName);
+    const origin1 = await page.waitForSelector(`[id="Talk to Customers"]`);
+    const origin2 = await page.waitForSelector(`[id="No Nicotine or Tobacco"]`);
+    const origin3 = await page.waitForSelector(`[id="3 Hours of Deep Work"]`);
+    const morning = await page.waitForSelector("#Morning");
+    const workDay = await page.waitForSelector("#Workday");
+    const evening = await page.waitForSelector("#Evening");
+    let ob = await origin1?.boundingBox();
+    // let ob2 = await origin2?.boundingBox(); // <-----start ||Currently not using this box as unless you make some sort of timer function, it will fail as it will try to drag the box before it has moved to the correct location.
```

Comment:
This can come out, right? ob is all we need - when the first item gets dragged to a new column, another one will take its place. 

Maybe we could have more descriptive variables for these items? like
optionsColumnItem1BoundingBox, morningColumnBoundingBox, workDayColumnBoundingBox, eveningColumnBoundingBox.  It's a little verbose but would help make it really clear what's happening.